### PR TITLE
feat(server): use cuda graph in logits warping

### PR DESCRIPTION
**Pull request by OlivierDehaene**

### CODE REVIEW

**Comment by njhill** [comment]
@OlivierDehaene this looks cool, do you know how it compares performance-wise to the vectorized impl in @jlamypoirier's PR? Could the two be combined?

**Comment by OlivierDehaene** [comment]
The performance in prod are not so great unfortunately.
I will investigate further tomorrow.
The two can be merged for sure
